gym/core.py

```diff
@@ -51,6 +51,7 @@ def __new__(cls, *args, **kwargs):
         env = super(Env, cls).__new__(cls)
         env._env_closer_id = env_closer.register(env)
         env._closed = False
+        env._spec = None
```

Comment:
Well there's currently a bug, that comes up pretty fast when you're not using registration. (And usually it doesn't make sense to use registration in the early stages of research, or when you're doing something outside of the standard single-task setting.) I'll add to the changelist and merge.